tests/cmd/git/commands.go

```diff
@@ -33,3 +39,36 @@ func Push(user model.User, keyPath string, app model.App, banner string) {
 	curlCmd = model.Cmd{CommandLineString: fmt.Sprintf(`curl -sL "%s"`, app.URL)}
 	Eventually(cmd.Retry(curlCmd, banner, cmdRetryTimeout)).Should(BeTrue())
 }
+
+// PushWithInterrupt executes a `git push deis master` from the current
+// directory using the provided key, but then halts the progress via SIGINT.
+func PushWithInterrupt(user model.User, keyPath string) {
+	sess := doGitPush(user, keyPath)
+	Eventually(sess.Err).Should(Say("Starting build... but first, coffee!"))
+
+	sess = sess.Interrupt()
```

Comment:
this is a bummer b/c it could introduce a timing dependency (the git push could complete before this executes)

I know there's no reasonable fix right now, so this is me just recording the thought. consider it the equivalent of the tiniest nit for now!